### ISSUE TITLE
VCS repo not installing throwing error

### ISSUE THREAD

**Issue by kishore-mohan** [rodjek/librarian-puppet#288]
In windows when try to install vcs repo module using Puppetfile. Its throwing error.

####PuppetFile#####

```
forge "https://forgeapi.puppetlabs.com"

# use dependencies defined in metadata.json
#metadata

mod 'puppetlabs/registry'
mod 'rismoney/chocolatey' 
mod 'puppetlabs/dism'
mod 'opentable/iis'
mod 'puppetlabs/vcsrepo
```

==> default: Installing Puppet modules in "puppet" with Librarian-Puppet...
C:/Users/user/.vagrant.d/gems/gems/librarian-puppet-1.4.1/lib/librarian/pup
pet/source/forge/repo.rb:65:in `install_version!': C:/chef/windows_vagrant_puppe
t/puppet/.tmp/librarian/cache/source/puppet/forge/forgeapi_puppetlabs_com/puppet
labs-vcsrepo/1.2.0/vcsrepo does not exist, something went wrong. Try removing it
 manually (Librarian::Error)

**Comment by carlossg** [comment]
yeah, there are reports of issues in windows, can you run rake in the project to run the tests and report the failures? I don't have access to a windows box

**Comment by elliotblackburn** [comment]
Hi @carlossg, I seem to be having the same issue as above but it seems this wasn't taken any further by OP. What would you like me to run to get you the information you need? I'm running this using [vagrant-librarian-puppet](https://github.com/voxpupuli/vagrant-librarian-puppet), i'm not sure if that makes much of a difference though.

**Comment by carlossg** [comment]
the project is now under https://github.com/voxpupuli/librarian-puppet

**Comment by elliotblackburn** [comment]
Ah sorry, my bad!
